tests/acceptance/features/apiSearchContent/contentSearch.feature

```diff
@@ -265,3 +265,39 @@ Feature: content search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario: space-admin checks the audio, image, photo and location properties of PROPFIND request of file from project space
```

Comment:
Why is this test in the search suite?
@saw-jan

Reply:
it needs tika service, we can have it in a separate feature file: `apiSearchContent/propfindExtractedProps.feature`